sockeye/output_handler.py

```diff
@@ -119,6 +121,54 @@ def handle(self,
         self.stream.flush()
 
 
+class NematusStringWithAlignmentsOutputHandler(StringOutputHandler):
+    """
+    Output handler to write translations and alignments to a stream. The format is compatible with the output
+    generated by Nematus. Note that unlike other output handlers each input sentence will result in an output
+    consisting of multiple lines.
+    More concretely the format is:
+
+    ```
+    sentence id ||| target words ||| score ||| source words ||| number of source words ||| number of target words
+    ALIGNMENT FOR T_1
+    ALIGNMENT FOR T_2
+    ...
+    ALIGNMENT FOR T_n
+    ```
+
+    where the alignment is a list of probabilities of alignment to the source words.
+
+    :param stream: Stream to write translations and alignments to.
+    """
+
+    def __init__(self, stream) -> None:
+        super().__init__(stream)
+
+    def handle(self,
+               t_input: sockeye.inference.TranslatorInput,
+               t_output: sockeye.inference.TranslatorOutput,
+               t_walltime: float = 0.):
+        """
+        :param t_input: Translator input.
+        :param t_output: Translator output.
+        :param t_walltime: Total wall-clock time for translation.
+        """
+        self.stream.write("%d ||| %s ||| %f ||| %s ||| %d ||| %d\n" % (t_input.id,
```

Comment:
I would prefer here a .format() call with more descriptive names. My rule of thumb is "more than 3 substitutions, use format()". But I won't force you, of course :-)

Reply:
sure, I can change this.